src/w2grid.js

```diff
@@ -713,6 +754,7 @@
             }
             var time = (new Date()).getTime();
             var obj = this;
+            var defaultToString = {}.toString;
```

Comment:
As I mentioned before, this is very confusing syntax for me. I am going to refactor it.